sdk/include/ddk/ntstrsafe.h

```diff
@@ -1674,77 +1801,92 @@ NTSTRSAFEAPI RtlStringCopyExWorkerA(STRSAFE_LPSTR pszDest,size_t cchDest,size_t
                     cchRemaining++;
                     Status = STATUS_BUFFER_OVERFLOW;
                 }
+
                 *pszDestEnd = '\0';
             }
         }
     }
+
     if (!NT_SUCCESS(Status))
     {
         if (pszDest)
         {
             if (dwFlags & STRSAFE_FILL_ON_FAILURE)
             {
-                memset(pszDest,STRSAFE_GET_FILL_PATTERN(dwFlags),cbDest);
-                if (STRSAFE_GET_FILL_PATTERN(dwFlags)==0)
+                memset(pszDest, STRSAFE_GET_FILL_PATTERN(dwFlags), cbDest);
+                if (STRSAFE_GET_FILL_PATTERN(dwFlags) == 0)
                 {
                     pszDestEnd = pszDest;
                     cchRemaining = cchDest;
                 }
-                else
-                    if (cchDest > 0)
-                    {
-                        pszDestEnd = pszDest + cchDest - 1;
-                        cchRemaining = 1;
-                        *pszDestEnd = '\0';
-                    }
-            }
-            if (dwFlags & (STRSAFE_NULL_ON_FAILURE | STRSAFE_NO_TRUNCATION))
-            {
-                if (cchDest > 0)
+                else if (cchDest > 0)
                 {
-                    pszDestEnd = pszDest;
-                    cchRemaining = cchDest;
+                    pszDestEnd = pszDest + cchDest - 1;
+                    cchRemaining = 1;
                     *pszDestEnd = '\0';
                 }
             }
+
+            if (dwFlags & (STRSAFE_NULL_ON_FAILURE | STRSAFE_NO_TRUNCATION) && cchDest > 0)
```

Comment:
Code style: surround the `dwFlags & (...)` by an extra pair of parentheses.

Reply:
Disagree. Or is that actually based on an official guideline we have somewhere?

Reply:
I remember having been told that some time ago....

Reply:
I've done that in my latest commit. Even if I may not fully agree with that, it's no reason to have another long discussion. And the previous style wasn't even unified, so something had to be done either way.